build.gradle

```diff
@@ -2,14 +2,14 @@
 
 buildscript {
     ext.supportLibraryVersion = '26.1.0'
-    ext.kotlin_version = '1.2.21'
+    ext.kotlin_version = '1.2.71'
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3"
+        classpath 'com.android.tools.build:gradle:3.2.0'
```

Comment:
build is failing because `3.2.0` requires build tools 28, but we are only including build tools 26 in the travis configs